test-utils/Makefile

```diff
@@ -5,7 +5,7 @@ help: ## Display this help screen
 		-E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | \
 		awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-test: $(SUBDIRS) ### Run the test command for all utils
+#test: $(SUBDIRS) ### Run the test command for all utils
```

Comment:
Why this was commented?

Reply:
Commenting this like out does not break anything and apparently it is not used anywhere. Since it would require additional efforts to port this to PLONK, the work should be done in another issue